tests/commons_test.sh

```diff
@@ -51,22 +72,20 @@ function testConfigFileNotFound
     true # Reset return value
 }
 
-function testConfigFileFound
+function testLocalConfigFile
 {
     . ./src/commons.sh --source-only
 
     declare -A expected_configurations=(
       [key1]="value1"
       [key2]="value2"
       [key3]="value3"
-      [ip]="127.0.0.1"
-      [port]="2222"
     )
 
     cp tests/samples/kworkflow.config tests/.tmp_commons_test/
 
     pushd tests/.tmp_commons_test
-    check_local_configuration
+    load_configuration
```

Comment:
I think we should remove pushd/popd and change the call to `load_configuration` here to `parse_configuration tests/.tmp_commons_test/kworkflow.config`. Why? `load_configuration` will try to load the global configuration`file, which is `$config_files_path/kworkflow.config`,  prior to local. Since `kw.sh` was not imported, `$config_files_path` equals to none. So it will try to load "/kworkflow.config". If the user testing kw have a local `kworkflow.config` at his/her root "/", them this test will fail. The test shouldn't depend on the user's environment to pass.

Reply:
Actually, I changed the test because it does not make sense anymore. Now, I'm testing the local config file with different parameters.